niworkflows/viz/utils.py

```diff
@@ -74,15 +79,15 @@ def svg_compress(image, compress='auto'):
     Performs compression (can be disabled). A bit hacky. '''
 
     # Compress the SVG file using SVGO
-    if (shutil.which("svgo") and compress == 'auto') or compress is True:
+    if (which("svgo") and compress == 'auto') or compress is True:
 
         p = subprocess.run("svgo -i - -o - -q -p 3 --pretty --disable=cleanupNumericValues",
                            input=image.encode('utf-8'), stdout=subprocess.PIPE,
                            shell=True, check=True)
         image = p.stdout.decode('utf-8')
```

Comment:
On the whole, I would prefer not to add dependencies. What about dropping `which` and just letting `subprocess` complain if the file is missing?

```Python
if compress:
    try:
        p = subprocess.run("svgo -i - -o - -q -p 3 --pretty --disable=cleanupNumericValues",
                           input=image.encode('utf-8'), stdout=subprocess.PIPE,
                           shell=True, check=True)
    except FileNotFoundError:
        if compress is True:
            raise
    else:
        image = p.stdout.decode('utf-8')
```

For `cwebp`, to avoid going into the parsing code, it might make sense to have something like:

```Python
def has_cwebp():
    try:
        subprocess.run(['cwebp'], stdout=subprocess.DEVNULL)
    except FileNotFoundError:
        return False
    return True
```